libraries/ms-common/inc/soft_timer.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <stdint.h>
+
+#include "status.h"
+
+#define SOFT_TIMER_NUM 10
+
+typedef void (*soft_timer_callback)(uint8_t timer_id, void* context);
```

Comment:
I would probably typedef the timer ID type just in case we want to change the size.